fe1-web/src/features/social/components/ChirpCard.tsx

```diff
@@ -155,6 +155,26 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
 
   const [showDeleteConfirmation, setShowDeleteConfirmation] = useState(false);
 
+  const thumbsPressed = (isThumbsUp: boolean) => {
+    const reactionToAdd = isThumbsUp ? '👍' : '👎';
+    const reactionToRemove = isThumbsUp ? '👎' : '👍';
+
+    if (reacted[reactionToAdd]) {
+      deleteReaction(reacted[reactionToAdd].id);
+      return;
+    }
+
+    // If the user has already reacted with the other reaction, remove it
+    if (reacted[reactionToRemove]) {
+      deleteReaction(reacted[reactionToRemove].id);
+    }
+
+    addReaction(reactionToAdd);
+  };
+
+  const thumbsUpPressed = () => thumbsPressed(true);
+  const thumbsDownPressed = () => thumbsPressed(false);
```

Comment:
If you need to pass an argument to a function feel free to write

`onPress={() => thumbsPressed("👍")}`

in the component directly. If there are no arguments, `onPress={someFunction}` is usually preferred.